packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -16,56 +24,82 @@ const Separator = styled(Toolbar.Separator)`
   height: ${pxToRem(24)};
 `;
 
-const ModifierButton = ({ icon, name, label }) => {
-  const editor = useSlate();
-
-  /**
-   * @param {string} name - name of the modifier
-   */
-  const isModifierActive = (name) => {
-    const modifiers = Editor.marks(editor);
-
-    return modifiers ? modifiers[name] === true : false;
-  };
-
-  /**
-   * @param {string} name - name of the modifier
-   */
-  const toggleModifier = (name) => {
-    const isActive = isModifierActive(name);
-
-    if (isActive) {
-      Editor.removeMark(editor, name);
-    } else {
-      Editor.addMark(editor, name, true);
-    }
-  };
-  const isActive = isModifierActive(name);
+const FlexButton = styled(Flex).attrs({ as: 'button' })`
+  &:hover {
+    background: ${({ theme }) => theme.colors.primary100};
+  }
+`;
 
+const ToolbarButton = ({ icon, name, label, isActive, handleClick }) => {
   const { formatMessage } = useIntl();
   const labelMessage = formatMessage(label);
 
   return (
     <Tooltip description={labelMessage}>
       <Toolbar.ToggleItem value={name} data-state={isActive ? 'on' : 'off'} asChild>
-        <Flex
+        <FlexButton
           background={isActive ? 'primary100' : ''}
-          padding={2}
-          as="button"
+          alignItems="center"
+          justifyContent="center"
+          width={7}
+          height={7}
           hasRadius
           onMouseDown={(e) => {
```

Comment:
That's true, but I'm not sure it's an issue for us here. If you're able to focus the toolbar buttons with your keyboard, doesn't it mean your editor is not focused, and therefore you have no selection to apply the modifiers to?

For context the reason I'm using onMouseDown is because in combination with e.preventDefault, it allows us to not lose the focus on the editor when clicking it. There could also be other workarounds, but that's how [Slate do it in their examples](https://github.com/ianstormtaylor/slate/blob/main/site/examples/richtext.tsx#L210).

In my opinion we should not let this block the PR for now and focus more on having good a11y after the alpha